app/src/main/java/org/simple/clinic/summary/SummaryListAdapterIds.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.summary
+
+object SummaryListAdapterIds {
+  const val PRESCRIBED_DRUGS = -1L
+  const val ADD_PRESCRIPTION = -2L
+  const val ADD_NEW_BP = -3L
```

Comment:
Considering that these are getting statically imported, do you think we should add an `ID_` prefix? This is also fine.

Reply:
The reason I didn't do it was because the enclosing object already has an `Ids` in the name.